src/Container/QueueFactory.php

```diff
@@ -163,8 +163,6 @@ public function __invoke(ContainerInterface $container): Queue
                 $exchangeObjects[$exchange] = ExchangeFactory::$exchange($container, $this->channel, true);
             }
 
-            $queue->declareQueue();
```

Comment:
This would potentially fail, if auto_setup_fabric is true, since you would bind a queue to an exchange which is not declared.